### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD042.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#44] [General]
# https://blog.3place.cn/3place/fsd/FSD042.html

https://blog.3place.cn/3place/fsd/FSD042.html

**Comment by picklecai** [comment]
虽然富足，却饥肠辘辘；

富而不足，原来双重意义：心感不足，真的不足（点石成金反而吃不到饭）。

**Comment by picklecai** [comment]
先失明，后面的误解才用得上。
